polaris/viz/__init__.py

```diff
@@ -121,18 +132,23 @@ def plot_horiz_field(ds, ds_mesh, field_name, out_file_name,
     legend_width = fig_width / 5
     figsize = (fig_width + legend_width, fig_width / aspect_ratio)
 
-    plt.figure(figsize=figsize)
-    ax = plt.subplot(111)
+    if create_fig:
+        plt.figure(figsize=figsize)
+        ax = plt.subplot(111)
     ax.add_collection(ocean_patches)
     ax.set_xlabel('x (km)')
     ax.set_ylabel('y (km)')
     ax.set_aspect('equal')
     ax.autoscale(tight=True)
-    plt.colorbar(ocean_patches, extend='both', shrink=0.7)
-    plt.title(title)
-    plt.tight_layout(pad=0.5)
-    plt.savefig(out_file_name)
-    plt.close()
+    cbar = plt.colorbar(ocean_patches, extend='both', shrink=0.7, ax=ax,
+                        pad=0.1)
+    if cmap_title is not None:
+        cbar.ax.set_title(cmap_title)
```

Comment:
```suggestion
        cbar.set_label(cmap_title)
```

Reply:
@cbegeman, This would place the colorbar label vertically, allowing for longer labels without overlap. I made this change in the inertial gravity wave case, but can revert it if you prefer `set_title`.